coderd/coderd.go

```diff
@@ -1659,6 +1661,13 @@ func (api *API) Close() error {
 	_ = api.AppSigningKeyCache.Close()
 	_ = api.AppEncryptionKeyCache.Close()
 	_ = api.UpdatesProvider.Close()
+
+	if current := api.PrebuildsReconciler.Load(); current != nil {
+		ctx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, xerrors.New("gave up waiting for reconciler to stop"))
```

Comment:
Maybe differentiate this from the cause when entitlements change?

```suggestion
		ctx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, xerrors.New("gave up waiting for reconciler to stop before shutdown"))
```

Reply:
Great idea 👍